live/dash/live_dash.py

```
@@ -459,6 +459,253 @@ def get_state():
+
+    if target_db.exists() and target_db.stat().st_size > 0:
+        archive_dir = target_db.parent / "archive"
+        archive_dir.mkdir(parents=True, exist_ok=True)
+        ts_str = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
+        archive_path = archive_dir / f"live_{ts_str}.db"
+        shutil.copy2(target_db, archive_path)
+        archived_name = archive_path.name
+        try:
+            target_db.unlink()
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)